src/client.ts

```diff
@@ -87,13 +76,11 @@ export class SubscriptionClient {
     this.connectionParams = connectionParams;
     this.connectionCallback = connectionCallback;
     this.url = url;
-    this.subscriptions = {};
-    this.maxId = 0;
-    this.subscriptionTimeout = timeout;
-    this.waitingSubscriptions = {};
+    this.requests = {};
```

Comment:
These should be called `operations`. Request is a one-time thing in most definitions, operation indicates that it can be an ongoing thing. It also makes sense, because query, mutation and subscription are all called "operations" in GraphQL.